tests/test_cbox.py

```diff
@@ -73,6 +74,55 @@ def hello(name):
     assert msg == ['test']
 
 
+def test_cmd_with_default_true_bool_argument():
```

Comment:
this test and `test_cmd_with_default_false_bool_argument` can be `pytest.mark.parameterize` tests instead of duplicated code.